docker: implement docker and docker-compose integration (#58)
raven-os/build-raven-os-org: builder/README.md

```bash
$ npm start
```

#### Docker

> Bind a docker daemon to the builder so he can run nbuild container,

Reviewer: ```suggestion
Bind the docker daemon's socket to the builder so it can run an nbuild container.
```

> it also needs two volumes, for manifests and packages and needs to be a tty

Reviewer: Note that it 
```suggestion
Note that it needs two volumes: one for the manifests and the built packages. It also needs to be a tty and an access to the internet.
```

```sh

> docker run -v /var/run/docker.sock:/var/run/docker.sock -v nbuild_manifests:/home/node/builder/manifests/ -v nbuild_out:/home/node/builder/out/ --net=host -t tmp/builder

Reviewer: Why do you need --net=host? It actually exposes the host network to the container, which is kinda meh :/

(also, edit the path to use /app instead of /home)
